tests/api/test_projects.py

```diff
@@ -220,16 +252,26 @@ def test_success(self, flask_api_url, pytestconfig):
         print(post_response.text)
         for id in post_response.json():
             assert UUID(id, version=4)
+        project_id = post_response.json()[0]
```

Comment:
this line is unnecessary, we should instead assert post_response.status_code == 200